style.css

```diff
@@ -174,6 +179,10 @@ a {
   color: rgb(248, 246, 246);
   text-decoration: none;
   display: inline-block;
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Kindly fix the popup windows page on the Desktop view, have a look at the recording screen video 👇🏻Actually almost every thing need to be improved 👍🏻  

[Screencast from 10-21-2022 06:52:23 PM.webm](https://user-images.githubusercontent.com/79058364/197250030-dab123ef-afb2-4e2c-91d2-2460d299cebe.webm)

Reply:
Hello @BenMukebo I test the changes in two different machines and I did not see the errors you show in the video, how can I check to be sure that my changes are ok? thank you